src/master/allocator/mesos/hierarchical.cpp

```diff
@@ -16,6 +16,7 @@
 
 #include "master/allocator/mesos/hierarchical.hpp"
 
+#include <iostream>
```

Comment:
I'm not sure why we need this?

Reply:
I will remove it, I added that for log purpose